### PR TITLE
add `make_gaussian`

### PR DESCRIPTION
**Pull request by Shoeboxam**

### CODE REVIEW

**Comment by Shoeboxam** [comment]
Current dependencies on/for this PR:
* main
  * **PR #696** <a href="https://app.graphite.dev/github/pr/opendp/opendp/696" target="_blank">Graphite</a> 
    * **PR #645** <a href="https://app.graphite.dev/github/pr/opendp/opendp/645" target="_blank">Graphite</a> 
      * **PR #691** <a href="https://app.graphite.dev/github/pr/opendp/opendp/691" target="_blank">Graphite</a> 
      * **PR #604** <a href="https://app.graphite.dev/github/pr/opendp/opendp/604" target="_blank">Graphite</a> 
        * **PR #706** <a href="https://app.graphite.dev/github/pr/opendp/opendp/706" target="_blank">Graphite</a> 
        * **PR #699** <a href="https://app.graphite.dev/github/pr/opendp/opendp/699" target="_blank">Graphite</a> 
        * **PR #687** <a href="https://app.graphite.dev/github/pr/opendp/opendp/687" target="_blank">Graphite</a> 
          * **PR #688** <a href="https://app.graphite.dev/github/pr/opendp/opendp/688" target="_blank">Graphite</a> 
            * **PR #689** <a href="https://app.graphite.dev/github/pr/opendp/opendp/689" target="_blank">Graphite</a> 
              * **PR #749** <a href="https://app.graphite.dev/github/pr/opendp/opendp/749" target="_blank">Graphite</a> 
              * **PR #712** <a href="https://app.graphite.dev/github/pr/opendp/opendp/712" target="_blank">Graphite</a> 
                * **PR #713** <a href="https://app.graphite.dev/github/pr/opendp/opendp/713" target="_blank">Graphite</a> 
                  * **PR #736** <a href="https://app.graphite.dev/github/pr/opendp/opendp/736" target="_blank">Graphite</a> 
                    * **PR #761** <a href="https://app.graphite.dev/github/pr/opendp/opendp/761" target="_blank">Graphite</a> 
                      * **PR #798** <a href="https://app.graphite.dev/github/pr/opendp/opendp/798" target="_blank">Graphite</a> 
                        * **PR #799** <a href="https://app.graphite.dev/github/pr/opendp/opendp/799" target="_blank">Graphite</a> 
                          * **PR #800** <a href="https://app.graphite.dev/github/pr/opendp/opendp/800" target="_blank">Graphite</a>  👈
                      * **PR #796** <a href="https://app.graphite.dev/github/pr/opendp/opendp/796" target="_blank">Graphite</a> 
              * **PR #690** <a href="https://app.graphite.dev/github/pr/opendp/opendp/690" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opendp/opendp/800?utm_source=stack-comment).

**Review by andrewvyrros — COMMENTED** [review]
Good stuff! This and #736 really do a lot to simply life for the user.

File: rust/src/measurements/gaussian/mod.rs [hunk 1]
```diff
@@ -86,67 +110,59 @@ where
 ///
 /// # Generics
 /// * `D` - Domain of the data type to be privatized. Valid values are `VectorDomain<AtomDomain<T>>` or `AtomDomain<T>`.
-/// * `MO` - Output Measure. The only valid measure is `ZeroConcentratedDivergence<T>`.
+/// * `MO` - Output measure. The only valid measure is `ZeroConcentratedDivergence<QO>`, but QO can be any float.
 /// * `QI` - Input distance. The type of sensitivities. Can be any integer or float.
-pub fn make_base_gaussian<D, MO>(
+pub fn make_gaussian<D: MakeGaussian<MO, QI>, MO: 'static + Measure, QI: 'static>(
```
[the remark this answers is not in the document]
Comment (Shoeboxam) [reply rust/src/measurements/gaussian/mod.rs]: I've been going by this precedence:
https://github.com/opendp/opendp/blob/main/rust/src/core/mod.rs#L20-L21

[the remark this answers is not in the document]
Comment (Shoeboxam) [reply rust/src/measurements/gaussian/mod.rs]: Should we change this order?

[the remark this answers is not in the document]
Comment (andrewvyrros) [reply rust/src/measurements/gaussian/mod.rs]: Nope, you're totally right, order is correct.

**Review by andrewvyrros — APPROVED** [review]

**Pull request merged**